.github/workflows/main.yaml

```
@@ -35,14 +35,18 @@ jobs:
-      - name: "🧪 Verify commit signature"
-        run: |
-          git verify-commit --verbose HEAD
 
       - name: "🧪 Verify index.html.sig"
         run: |
           gpg --verify --with-fingerprint index.html.sig index.html
+
+      - name: "🔑 Receive GitHub Key"
+        run: |
+          gpg --recv-key B5690EEEBB952194
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)